### PR TITLE
nixos: add freedesktop/gnome/myself maintainers

### PR DESCRIPTION
**Pull request by worldofpeace**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by jonringer — APPROVED** [review]
LGTM

assuming ofborg checkmeta passes

**Comment by worldofpeace** [comment]
@jonringer Does it eval drafts? (can't remember, been a while since I did a draft)

**Comment by worldofpeace** [comment]
Guess not.

**Comment by jonringer** [comment]
@GrahamcOfBorg eval

**Comment by jonringer** [comment]
ofborg has been touchy recently.

**Comment by jonringer** [comment]
ah, see https://github.com/NixOS/nixpkgs/pull/84058#issuecomment-607555664

File: nixos/modules/programs/nm-applet.nix [hunk 1]
```diff
@@ -1,6 +1,10 @@
 { config, lib, pkgs, ... }:
 
 {
+  meta = {
+    maintainers = teams.freedesktop.members;
```
Comment (jtojnar) [inline nixos/modules/programs/nm-applet.nix:5]:
```suggestion
    maintainers = lib.teams.freedesktop.members;
```

File: nixos/modules/services/desktops/geoclue2.nix [hunk 2]
```diff
@@ -60,6 +60,10 @@ let
 in
 {
 
+  meta = {
+    maintainers = with maintainers; [ worldofpeace ];
+  };
+
```
Comment (jonringer) [inline nixos/modules/services/desktops/geoclue2.nix:66]: already defined
```
attribute 'meta.maintainers' at /home/jon/.cache/nixpkgs-review/pr-84059/nixpkgs/nixos/modules/services/desktops/geoclue2.nix:271:3 already defined at /home/jon/.cache/nixpkgs-review/pr-84059/nixpkgs/nixos/modules/services/desktops/geoclue2.nix:64:5
```

**Comment by jonringer** [comment]
you can also do meta check locally if you follow https://github.com/NixOS/ofborg#running-meta-checks-locally

**Comment by worldofpeace** [comment]
> you can also do meta check locally if you follow https://github.com/NixOS/ofborg#running-meta-checks-locally

hehe, that thing uses way too much memory.

**Review by jonringer — APPROVED** [review]

**Comment by worldofpeace** [comment]
Thanks @jonringer

**Pull request merged**